micrometer-tracing/src/main/java/io/micrometer/tracing/exporter/FinishedSpan.java

```diff
@@ -60,6 +62,13 @@ public interface FinishedSpan {
      */
     Instant getEndTimestamp();
 
+    /**
+     * @return span's duration
+     */
+    default Duration getDuration() {
```

Comment:
Could you please add two tests for this method (one for Brave, one for OTel)?
Like these:
- `BraveFinishedSpanTests#should_calculate_instant_from_brave_timestamps`
- `OtelFinishedSpanTests#should_calculate_instant_from_otel_timestamps`

Reply:
test added.